pure.zsh

```diff
@@ -261,6 +271,13 @@ prompt_pure_setup() {
 	zstyle ':vcs_info:git*' formats ' %b'
 	zstyle ':vcs_info:git*' actionformats ' %b|%a'
 
+	# if the user has not registered a custom zle widget for clear-screen,
+	# override the builtin one so that the preprompt is displayed correctly when
+	# ^L is issued.
+	if [[ $widgets[clear-screen] == "builtin" ]]; then
```

Comment:
Actually, this doesn't even need quotes, but I usually do it anyway just to indicate what it is :/.